src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Queries/GetWorkplacesQuery.g.cs

```diff
@@ -8,33 +8,64 @@
 using Nox.Application.Commands;
 
 using ClientApi.Application.Dto;
-using ClientApi.Infrastructure.Persistence;
+using ClientApi.Infrastructure.Persistence;using Nox.Presentation.Api;
+using Nox.Solution;
 
 namespace ClientApi.Application.Queries;
 
 public record GetWorkplacesQuery() : IRequest<IQueryable<WorkplaceDto>>;
 
 internal partial class GetWorkplacesQueryHandler: GetWorkplacesQueryHandlerBase
 {
-    public GetWorkplacesQueryHandler(DtoDbContext dataDbContext): base(dataDbContext)
+    public GetWorkplacesQueryHandler(DtoDbContext dataDbContext,
+        NoxSolution solution,
+        IHttpLanguageProvider languageProvider): base(dataDbContext,
+            solution,
+            languageProvider)
     {
     
     }
 }
 
 internal abstract class GetWorkplacesQueryHandlerBase : QueryBase<IQueryable<WorkplaceDto>>, IRequestHandler<GetWorkplacesQuery, IQueryable<WorkplaceDto>>
-{
-    public  GetWorkplacesQueryHandlerBase(DtoDbContext dataDbContext)
+{private readonly NoxSolution _solution;
+        private readonly IHttpLanguageProvider _languageProvider;
+
+    public  GetWorkplacesQueryHandlerBase(DtoDbContext dataDbContext,
+        NoxSolution solution,
+        IHttpLanguageProvider languageProvider)
     {
-        DataDbContext = dataDbContext;
+        DataDbContext = dataDbContext;_solution = solution;
+        _languageProvider = languageProvider;
     }
 
     public DtoDbContext DataDbContext { get; }
 
     public virtual Task<IQueryable<WorkplaceDto>> Handle(GetWorkplacesQuery request, CancellationToken cancellationToken)
     {
-        var item = (IQueryable<WorkplaceDto>)DataDbContext.Workplaces
-            .AsNoTracking();
-       return Task.FromResult(OnResponse(item));
+        var cultureCode = _languageProvider.GetLanguage();
+
+        IQueryable<WorkplaceDto> linqQueryBuilder =
+            from item in DataDbContext.Workplaces.AsNoTracking()
+            join itemLocalizedFromJoin in DataDbContext.WorkplacesLocalized on cultureCode equals itemLocalizedFromJoin.CultureCode into joinedData
```

Comment:
```suggestion
            join itemLocalizedFromJoin in DataDbContext.WorkplacesLocalized on cultureCode equals '###CULTURECODE###' into joinedData
```

-- @__cultureCode_0='en-US'
SELECT w."Id", w."Name", COALESCE(t."Description", ('[' || COALESCE(w."Description", '')) || ']') AS "Description", w."Greeting", w."BelongsToCountryId", w."Etag"
FROM "Workplaces" AS w
LEFT JOIN (
    SELECT w0."Id", w0."Description"
    FROM "WorkplacesLocalized" AS w0
    WHERE 'en-US' = w0."CultureCode"
) AS t ON w."Id" = t."Id"

Reply:
Let's discuss on call